### ISSUE TITLE
Python 3.12 error.

### ISSUE THREAD

**Issue by mafudge** [mdeakyne/BbRest#29]
<string>:1: SyntaxWarning: invalid escape sequence '\['

**Comment by eflage** [comment]
This was introduced in Python 3.12: https://docs.python.org/dev/whatsnew/3.12.html#other-language-changes. Seems like the correct way to fix it is to use raw strings in all the regular expressions.

**Comment by mafudge** [comment]
likely a dependency issue? Seem like the **bbrest.py** uses raw strings.

**Comment by eflage** [comment]
I agree. Looking at the imports, it looks like maya might be the culprit. https://github.com/kennethreitz/maya/issues/202
